packages/cli/src/lib/deploy/PolywrapDeploy.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { Deployer } from "./deployer";
+import { loadDeployManifest, loadDeployManifestExt } from "../project";
+import { CacheDirectory } from "../CacheDirectory";
+import { Logger } from "../logging";
+
+import { DeployManifest } from "@polywrap/polywrap-manifest-types-js";
+import { Schema as JsonSchema } from "jsonschema";
+import path from "path";
+import nodePath from "path";
+
+interface PolywrapDeployConfig {
+  cache: CacheDirectory;
+  logger: Logger;
+  defaultModulesCached: boolean;
+}
+
+export class PolywrapDeploy {
+  public static cacheLayout = {
```

Comment:
not really, just following the same pattern as `PolywrapProject` - I also think that makes sense since these are constant- maybe adding a `readonly` would be enough?

Reply:
yeah, that makes sense!